landingPage/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JatayU.I</title>
+<link rel="stylesheet" href="landingPage.css">
+<script defer src="https://friconix.com/cdn/friconix.js"> </script>
+</head>
+<body>
+    
+    <nav class="navigation-menu">
+
+<div class="left-side-nav"> 
+
+<img class="logo-component" src="images\logo.PNG" alt="logo image">
+<a class="heading" href="http://">JatayU.I</a>
+
+</div>
```

Comment:
You can do proper code formatting by using ctrl shift p ,if you have downloaded the extension for that in vs code,to show the indentation and nesting.
Also img tag can be used inside a tag if you want to make it a link also like for home,not necessary.